### PR TITLE
python3Packages.async-upnp-client: init at 0.14.14

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

Enables the UPNP and DLNA components in home-assistant.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/async-upnp-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-asyncio
+    pytestcov
+  ];
+
+  checkPhase = ''
+    py.test --cov=async_upnp_client --cov-report=term
```
Comment (jonringer) [inline pkgs/development/python-modules/async-upnp-client/default.nix:32]: coverage is great to run upstream, but doesn't provide a lot of value for asserting package validity. However, these are cheap to run, so I don't mind including them.

Personally, I wouldn't worry about enabling code coverage for nixpkgs :).

**Review by jonringer — COMMENTED** [review]
the home-assistant changes should probably go into a different commit, as they aren't directly related to the python package additions

File: pkgs/development/python-modules/async-upnp-client/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/async-upnp-client/default.nix:32]: Thanks for the feedback regarding cov, I will drop them.

**Comment by mweinelt** [comment]
Updated.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 94487 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>59 packages marked as broken and skipped:</summary>  - clang-sierraHack-stdenv  - crossLibcStdenv  - digitalbitbox  - iterm2  - linuxPackages-libre.sch_cake  - linuxPackages.sch_cake  - linuxPackages_4_14.sch_cake  - linuxPackages_4_4.evdi  - linuxPackages_4_4.sysdig  - linuxPackages_4_4.zenpower  - linuxPackages_4_9.sysdig  - linuxPackages_4_9.zenpower  - linuxPackages_5_6.sch_cake  - linuxPackages_5_7.sch_cake  - linuxPackages_hardkernel_4_14.bcc  - linuxPackages_hardkernel_4_14.bpftrace  - linuxPackages_hardkernel_4_14.can-isotp  - linuxPackages_hardkernel_4_14.chipsec  - linuxPackages_hardkernel_4_14.digimend  - linuxPackages_hardkernel_4_14.evdi  - linuxPackages_hardkernel_4_14.mba6x_bl  - linuxPackages_hardkernel_4_14.nvidia_x11  - linuxPackages_hardkernel_4_14.nvidiabl  - linuxPackages_hardkernel_4_14.prl-tools  - linuxPackages_hardkernel_4_14.r8125  - linuxPackages_hardkernel_4_14.r8168  - linuxPackages_hardkernel_4_14.rtl8814au  - linuxPackages_hardkernel_4_14.rtl8821au  - linuxPackages_hardkernel_4_14.rtl8821ce  - linuxPackages_hardkernel_4_14.rtl88xxau-aircrack  - linuxPackages_hardkernel_4_14.rtlwifi_new  - linuxPackages_hardkernel_4_14.sch_cake  - linuxPackages_hardkernel_4_14.sysdig  - linuxPackages_hardkernel_4_14.vhba  - linuxPackages_hardkernel_4_14.virtualbox  - linuxPackages_hardkernel_4_14.virtualboxGuestAdditions  - linuxPackages_hardkernel_4_14.wireguard  - linuxPackages_hardkernel_4_14.xpadneo  - linuxPackages_hardkernel_4_14.zenpower  - linuxPackages_latest_xen_dom0.sch_cake  - linuxPackages_latest_xen_dom0_hardened.sch_cake  - linuxPackages_xen_dom0_hardened.sch_cake  - newlibCross  - ocamlPackages.stog  - php72Packages.phpmd  - php73Extensions.pcs  - php73Extensions.zmq  - php73Packages.phpmd  - php74Extensions.pcs  - php74Extensions.pthreads  - python27Packages.caffe  - python38Packages.libselinux  - python38Packages.nixpart  - python38Packages.pyblock  - steam-run-native  - tqsl  - uclibcCross  - ucommon  - xpdf</details><details>  <summary>4 packages built:</summary>  - python37Packages.async-upnp-client  - python37Packages.python-didl-lite  - python38Packages.async-upnp-client  - python38Packages.python-didl-lite</details>

**Pull request merged**